pkg/solana/txm/pendingtx.go

```diff
@@ -563,6 +576,52 @@ func (c *pendingTxContext) TrimFinalizedErroredTxs() int {
 	return len(expiredIDs)
 }
 
+func (c *pendingTxContext) IsTxReorged(sig solana.Signature, sigOnChainState txmutils.TxState) (string, bool) {
```

Comment:
Might be useful to add some comments above these new functions

Reply:
Do you think I should duplicate the comment? I put it at the [interface level](https://github.com/smartcontractkit/chainlink-solana/blob/ea4d0fa7633b7a714c38ddf38b1872f61c6dbfe5/pkg/solana/txm/pendingtx.go#L54-L59) as most of them are placed there

Reply:
Ah, no that's fine sorry I missed that!